scripts/build-shrink-ray.js

```diff
@@ -1,73 +1,95 @@
-/* eslint-disable no-console */
-
-require('isomorphic-fetch');
```

Comment:
Doesn't look like this is used anywhere else, can we remove it from the package.json?